doc/manuals/admin/database_model.md

```diff
@@ -307,6 +307,10 @@ Fields:
 -   **qs**: optional field to store the query parameters keymap for notification customization functionality in NGSIv2.
 -   **method**: optional field to store the HTTP method for notification customization functionality in NGSIv2.
 -   **payload**: optional field to store the payload for notification customization functionality in NGSIv2.
+-   **lastFailure**: the time when last notification failure occurred.
```

Comment:
Commit d8c932f have removed the whole sentence. However, I think it could be better to use:

"Not present if the subscription is OK and it has never failed".